src/utils/fetchData.js

```diff
@@ -0,0 +1,36 @@
+import { collection, query } from 'firebase/firestore';
+import { db } from './firebase';
+import { useFirestoreQuery } from '@react-query-firebase/firestore';
+
+export const fetchEntries = () => {
+  const ref = query(collection(db, 'Entries'));
+  const entriesQuery = useFirestoreQuery(['Entries'], ref);
```

Comment:
pass `{subscribe: true}` as a second arg for real-time updates, but we don't need it to be real-time update..